src/style.css

```
@@ -1 +1,131 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+body {
+  padding: 0;
+}
+
+.search-panel {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  height: 70px;
+  width: 100%;
+  box-sizing: border-box;
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: rgb(255, 255, 255);
+  box-shadow: 1px 1px 8px 1px #3d4e611a;
+  display: flex;
+  align-items: center;
+  outline: none;
+
+}
+
+.search-bar-2 {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  padding-left: 33px;
+  height: 42px;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid rgb(226, 232, 238);
+  background: rgb(255, 255, 255);
+  outline: none;
+
+}
+
+.search-panel-small {
+  position: relative;
+  margin-top: 20px;
+
+}
+
+.search-bar:hover {
+  border-radius: 4px;
+  border: 1px solid rgb(224, 231, 237);
+  background: rgb(255, 255, 255);
+  box-shadow: 1px 3px 8px 1px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
+  border-radius: 3px;
+  border: 1px solid rgb(225, 231, 237);
+  background: linear-gradient(180deg, rgb(255, 255, 255) 0%,
+  rgb(247, 247, 247) 100%);
+  box-shadow: 1px 4px 4px 1px rgba(0, 0, 0, 0.25);
+}
+
+.search-bar-2:hover {
+border-radius: 4px;
+border: 1px solid #E1E7ED;
+background: #FFF;
+box-shadow: 1px 3px 8px 1px rgba(61, 78, 97, 0.20);
+}
```

```suggestion
.search-bar-small:hover {
  border-radius: 4px;
  border: 1px solid #E1E7ED;
  background: #FFF;
  box-shadow: 1px 3px 8px 1px rgba(61, 78, 97, 0.20);
}
```
pay attention to indentation (according to code-style rules)

also, it's better to use descriptive names for classes - `small` is better than `2`